tensorflow/python/compiler/tensorrt/test/vgg_block_test.py

```diff
@@ -60,7 +60,7 @@ def ExpectedEnginesToBuild(self, run_params):
 
   # TODO(b/159459919): remove this routine to disallow native segment execution.
   def setUp(self):
-    super(trt_test.TfTrtIntegrationTestBase, self).setUp()  # pylint: disable=bad-super-call
+    super(VGGBlockTest, self).setUp()  # pylint: disable=bad-super-call
```

Comment:
Shouldn't this be `super().setUp()` according to [pep-3135](https://peps.python.org/pep-3135/)?

Reply:
Please also remove # pylint.

Reply:
@bixia1 fixed
closing